tests/secrets/test_local_filesystem.py

```diff
@@ -104,6 +105,19 @@ def test_missing_file(self, mock_exists):
         ):
             local_filesystem.load_variables("a.json")
 
+    @parameterized.expand(
+        (
+            ({}, {}),
+            ({"KEY": "AAA"}, {"KEY": "AAA"}),
+            ({"KEY_A": "AAA", "KEY_B": "BBB"}, {"KEY_A": "AAA", "KEY_B": "BBB"}),
```

Comment:
Can you use YAML file here?

Reply:
Added  YAML files for both variables and connection tests